src/bp/sdk/botpress.d.ts

```diff
@@ -1363,7 +1363,7 @@ declare module 'botpress/sdk' {
   export interface FormData {
     id?: string
     contentType?: string
-    [key: string]: undefined | number | boolean | string | FormData[]
+    [key: string]: FormData[] | any
```

Comment:
no real reason to have FormData[] if there's any

Reply:
I find it useful to know that this particular type can be used, I just find listing all builtin types to be a bit too much (would also need to add string[], and probably others we forgot).

What do you think?

Reply:
yeah it makes sense, just saying that having `FormData[] | any` is equivalent to `any` so might as well just put that, no?